cmd/cloud/deployment.go

```diff
@@ -325,6 +334,64 @@ func newDeploymentVariableUpdateCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+//nolint:dupl
+func newDeploymentHibernateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "hibernate [DEPLOYMENT-ID]",
+		Aliases: []string{"hb"},
+		Short:   "Hibernate an Astro Deployment",
+		Long:    "Hibernate an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
nit: might be simpler to keep even this logic as part of `deploymentOverrideHibernation` itself, since it is logically the same for both `newDeploymentHibernateCmd` and `newDeploymentWakeUpCmd` commands.

It will also simplify the flag handling, as of now some flags are being passed explicitly vs some of them are being passed implicitly to `deploymentOverrideHibernation` function

Reply:
Thanks, I wanted to highlight that resuming the schedules uses a shortcut when calling hibernate/wake, but will think again about refactoring this.